### PR TITLE
Added function to make the node features

### PR DESCRIPTION
**Pull request by AugustoPeres**
Adds a function for node features. At the moment this is just the wind vector.

### CODE REVIEW

File: meshnets/data_processing/data_mappers.py [hunk 1]
```diff
@@ -54,3 +54,19 @@ def make_edge_features(example):
...
+def make_node_features(example):
+    """Makes the node features from the node coordinates.
+
+    The node features:
+
+    - The wind vector. This is a common value for all the nodes in the
+      graph.
+
+    """
+    nodes = np.array(example['nodes'])
```
Comment (hpenedones) [inline meshnets/data_processing/data_mappers.py:68]: why do you need to convert this to numpy? Seems like the only thing you need is the number of nodes to iterate on...  Can you do num_nodes = len(example['nodes']) or similar?

File: meshnets/data_processing/data_mappers.py [hunk 2]
```diff
@@ -54,3 +54,19 @@ def make_edge_features(example):
...
+def make_node_features(example):
+    """Makes the node features from the node coordinates.
+
+    The node features:
+
+    - The wind vector. This is a common value for all the nodes in the
+      graph.
+
+    """
+    nodes = example['nodes']
```
Comment (IvanPombo) [inline meshnets/data_processing/data_mappers.py:68]: Remove nodes here.

**Review by IvanPombo — APPROVED** [review]
Otherwise it lgtm!

File: meshnets/data_processing/data_mappers.py [hunk 3]
```diff
@@ -54,3 +54,19 @@ def make_edge_features(example):
...
+    example['node_features'] = [
+        example['wind_vector'] for _ in range(len(nodes))
```
Comment (IvanPombo) [inline meshnets/data_processing/data_mappers.py:70]: If the purpose is to be general, change `wind_vector` with `feature`

**Pull request merged**